client/src/Components/CardContainer/index.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable operator-linebreak */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Empty, Pagination, Typography } from 'antd';
+
+import Card from '../Card';
+
+import './style.css';
+
+const { Title } = Typography;
+const cardContainer = ({ title, data, ...rest }) => {
+  const numEachPage = 8;
+  const [limit, setLimit] = useState([0, numEachPage]);
+
+  return (
+    <div className="container" {...rest}>
+      <Title id="container-title" level={2}>
+        {title}
+      </Title>
+      <div className="card-container">
+        {data && data.length !== 0 ? (
```

Comment:
rename the data prop name to providers => for more descriptive

Reply:
done